docs/source/tutorials/segmentation.ipynb

```diff
@@ -449,13 +449,13 @@
    "source": [
```

Comment:
need to resolve merge conflict too, think you made this PR from an outdated starting state